### DISCUSSION TITLE
Would it be possible to split the plugin into a stable and development branch?

### DISCUSSION THREAD

**Discussion by Qniol** [intro-skipper/intro-skipper#403] [Ideas]
Hey, I know that development is necessary. The only problem is that after every plugin update, the server must be restarted. With a dozen or so users, the functionality stopping working in the middle of the day causes the phone to go crazy. I don't know if it's just me who has this problem, but maybe it would be possible to split the repository into two, one with the main releases and the other with the development version for people who want the latest version as quickly as possible at the cost of having to restart frequently?

**Comment by AbandonedCart** [comment]
In your Dashboard, under Scheduled Tasks, is a task named Update Plugins. The default is to run on every start and every 24 hours. You can always change that. It is also possible to continue using the current version until a restart, unless the server version has updated and the current plugin is no longer compatible with the current server. In that case, though, I would think you'd want the update that restores functionality.

The plugin only updates about once a week unless there is legitimate reason to update sooner. These updates are typically significant feature improvements (adding recap / preview support) or significant to mitigate issues (removing injection unless it is specifically enabled, update to support changes to the server, etc). Preview versions are only published as manual installs. These would be the beta builds.

The problem with having an arbitrary release schedule is that we invest more time telling people to simply wait for a fix we already have that could be invested in the next fix or improvement. It also means that anyone looking for that fix has to either wait until it becomes a release or switch to a separate manifest, which then also has to be maintained.

TL;DR - Nobody is forcing you to update immediately. You have the option to simply wait.
